src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -191,6 +191,20 @@ object MiningConfig {
   }
 }
 
+trait VMConfig {
+  val maxCodeSize: Option[BigInt]
+}
+
+object VMConfig {
```

Comment:
I'm not sure about a completely new trait and config section. To me this belongs under `blockchain` (with all the fork block numbers) and should end up in `EvmConfig`.